### ISSUE TITLE
Error from array CSRF token

### ISSUE THREAD

**Issue by joshkel** [bcit-ci/CodeIgniter#5825]
If the user POSTs an array value for the CSRF token, then CodeIgniter generates a PHP warning while trying to validate the CSRF value:

```
Severity: Warning
Message: hash_equals(): Expected known_string to be a string, array given
Filename: core/Security.php
Line Number: 231
```

Since the POSTed CSRF token is under the control of the user, CodeIgniter should cleanly handle this, instead of generating a warning.

**Comment by narfbg** [comment]
This was already fixed in ba67b297d1e2baa281eed2cc7bf1fb16742cde87.

**Comment by joshkel** [comment]
Thank you.

The fix is not yet in the develop branch?
